src/features/identityCheck/components/ParentInformation.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { ParentInformationModal } from 'features/identityCheck/components/modals/ParentInformationModal'
+import { analytics } from 'libs/analytics'
+import { ButtonQuaternarySecondary } from 'ui/components/buttons/ButtonQuarternarySecondary'
+import { useModal } from 'ui/components/modals/useModal'
+import { ParentWithChild } from 'ui/svg/icons/ParentWithChild'
+import { PlainArrowNext } from 'ui/svg/icons/PlainArrowNext'
+import { Spacer, Typo, getSpacing } from 'ui/theme'
+
+export const ParentInformation: React.FC = () => {
+
+  return (
+    <GreyContainer>
+      <StyledParentWithChild />
+      <Spacer.Column numberOfSpaces={2} />
+      <StyledTitle>{'Accompagnement parental'}</StyledTitle>
```

Comment:
Pourquoi tu as besoin de faire {'text'} ?

Reply:
Sinon j'avais cette erreur qui respecte pourtant bien la règle : 
No raw text outside tags <Text>, <Typo.***>, <Styled***> or tag with prefix 'Text'. 
 *** = all exported Typo in src/ui/theme/typography.tsxeslint(local-rules/no-raw-text)